MANIFEST.in

```
@@ -0,0 +1,5 @@
+include benchmarks/reframe_config.py
+recursive-include benchmarks/apps *
+recursive-include benchmarks/examples *
+recursive-include benchmarks/modules *
+recursive-include benchmarks/spack *
```

```suggestion
recursive-include benchmarks/spack *

```
:smiling_face_with_tear: